pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -320,9 +332,14 @@ type ClusterWorkspaceTypeList struct {
 // ClusterWorkspaceInitializer is a unique string corresponding to a cluster workspace
 // initialization controller for the given type of workspaces.
 //
-// +kubebuilder:validation:Pattern:="^root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?)$"
+// TODO: Add tests for this regex
+// +kubebuilder:validation:Pattern:="^(root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?))|(system:.+)$"
```

Comment:
Note: This regex pattern is really only checked here: https://github.com/kcp-dev/kcp/blob/main/pkg/virtual/initializingworkspaces/builder/build.go#L391

I don't believe this change will cause a problem with the authorizer, since an error will result in `NoOpinion`, denying access.